.github/ISSUE_TEMPLATE/bug-swr-openapi.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "Bug report: swr-openapi"
```

Comment:
@htunnicliff Any changes/unique questions you’d like to ask for swr-openapi? (Ditto for feature requests)?